.github/workflows/verified-mods.yml

```diff
@@ -0,0 +1,11 @@
+name: VerifiedMods
+on: [push, pull_request]
+
+jobs:
+  manifest-checking:
+    name: Manifest check
+    uses: ./.github/workflows/manifest-check.yml
```

Comment:
Kinda surprised you have to pass the path of the YML file instead of being able to just call it by relative path or name directly... ^^

Reply:
You never know the exact context of execution for your workflows, especially with runners, so I feel like absolute paths are the way to go :)

Reply:
I mean we are only using GitHub's which should be well defined but I get your notion, haha ^^